src/supermarket/spec/api/cookbook_collaborator_criticism_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'spec_helper'
+
+describe 'GET /api/v1/cookbooks/:cookbook/collaborator' do
+  let(:cookbook) do
+    create(:cookbook, cookbook_versions: [
+      create(
+        :cookbook_version,
+        collaborator_failure: true,
+        collaborator_feedback: 'This cookbook does not have sufficient collaborators.'
+      )
+    ])
+  end
+
+  let(:cookbook_collaborator_signature) do
+    {
+      'failed' => cookbook.collaborator_failure,
+      'feedback' => cookbook.collaborator_feedback
+    }
+  end
+
+  it 'returns a 200' do
+    get "/api/v1/cookbooks/#{cookbook.name}/collaborator"
+
+    expect(response.status.to_i).to eql(200)
+  end
+
+  it 'returns the collaborator results' do
+    get "/api/v1/cookbooks/#{cookbook.name}/collaborator"
+
+    expect(signature(json_body)).to include(cookbook_collaborator_signature)
+  end
+end
```

Comment:
Move from having `foodcritic` and `collaborator` endpoints on a cookbook to a quality section within the endpoint for a cookbook version. Information about failures and quality for a given version of a cookbook should be housed in the api endpoint for a cookbook version: For example, within `/api/v1/cookbooks/dragons/versions/0.1.1` there would be a quality section of json.